src/test-gui/java/de/qabel/desktop/ui/accounting/item/AccountingItemGuiTest.java

```diff
@@ -35,9 +35,13 @@ protected FXMLView getView() {
     }
 
     @Test
+    public void testMenu() throws Exception {
+        page.openMenu();
+    }
+
+    @Test
     public void testEdit() throws Exception {
-        page.edit().inputAndConfirm("new alias");
-        waitUntil(() -> identity.getAlias().equals("new alias"));
+        page.edit();
```

Comment:
You should keep the assert statements inside the test. By looking at the test, I cannot say what is actually tests. A good to read test would be for example:
```JAVA
@Test
public void identityAliasIsEditable() throws Exception {
    page.editAlias("new alias");
    assertEquals("new alias", identity.getAlias());
}
```